lib/compilers/_all.js

```diff
@@ -41,6 +41,7 @@ export { GCCRSCompiler } from './gccrs';
 export { GolangCompiler } from './golang';
 export { HaskellCompiler } from './haskell';
 export { ISPCCompiler } from './ispc';
+export { KotlinCompiler } from './kotlin';
 export { JavaCompiler } from './java';
```

Comment:
These exports should be alphabetically ordered

```suggestion
export { ISPCCompiler } from './ispc';
export { JavaCompiler } from './java';
export { KotlinCompiler } from './kotlin';
```